src/routes/transactions/transactions.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { SafeRepository } from '../../domain/safe/safe.repository';
+import { ISafeRepository } from '../../domain/safe/safe.repository.interface';
+import { Page } from '../common/entities/page.entity';
+import { MultisigTransaction } from './entities/multisig-transaction.entity';
+import { MultisigTransactionMapper } from './mappers/multisig-transaction.mapper';
+
+@Injectable()
+export class TransactionsService {
+  constructor(
+    @Inject(ISafeRepository) private readonly safeRepository: SafeRepository,
+    private readonly mapper: MultisigTransactionMapper,
+  ) {}
+
+  async getMultisigTransactions(
+    chainId: string,
+    safeAddress: string,
+    executionDateGte?: string,
+    executionDateLte?: string,
+    to?: string,
+    value?: string,
+    nonce?: string,
+    executed?: boolean,
+    limit?: number,
+    offset?: number,
+  ): Promise<Partial<Page<MultisigTransaction>>> {
```

Comment:
This uses `Partial<T>` to comply with the current implementation, which does not return the `count` attribute in this route.